markdown-scheme_v3.Rmd

```diff
@@ -0,0 +1,393 @@
+            \begin{lstlisting}
+              coefplot, distinct, lincomeest, lprobust.ado
+            \end{lstlisting}
+          \item Instructions: Write: "See readme in reproducibility package"
+          \item Repository: ICPSR, Harvard Dataverse, Journal webpage etc. For ICPSR
+                the correct description is
+                \begin{lstlisting}
+                  AEA Data and Code Repository (ICPSR)
+                  Reproducible Research Repository (World Bank)
+                \end{lstlisting}
+          \item Notes: \textcolor{red}{Here who verified the package}. 
+            \begin{lstlisting}[breaklines]
+            Checked for reproducibility externally at the journal.
+            Checked for reproducibility at Development Impact Evaluation (DIME), World Bank
+            \end{lstlisting}
+          \item License:
+            \begin{itemize}
+              \item PP
+              \begin{itemize}
+                \item ICPSR:
+                  \begin{lstlisting}[breaklines]
+                  Creative Commons Attribution 4.0 International (CC BY 4.0) License
+                  https://creativecommons.org/licenses/by/4.0/
+                  \end{lstlisting}
+              \end{itemize}
+              \item WP: Modified BSD3
+            \end{itemize}
+      \end{enumerate}
+  \item Repository: Harvard, ICPSR- Type empty - URL, Journal webpage
+    \begin{lstlisting}[breaklines]
+        AEA Data and Code Repository
+        https://www.openicpsr.org/openicpsr/aea
+    \end{lstlisting}
+  \item Reproducibility:
+      \begin{enumerate}
+            \item Technology environment: empty
+            \item Technology Requirements: Standard, Cluster...; include runtime if available
+            \item Reproduction instructions: No details:
+                \begin{lstlisting}[breaklines]
+                 A readme file with detailed instructions is part of the (external) reproducibility package.
+                \end{lstlisting}
+                 Short details:
+                  \begin{lstlisting}
+                    A readme file and a single do file, AP_final_AEJ. do are provided. 
+                  \end{lstlisting}
+        \end{enumerate}
+\end{enumerate}
+
+## Access and Rights
+\begin{enumerate}
+    \item Disclaimer: \textcolor{red}{Perhaps not necessary}
+      \begin{lstlisting}[breaklines]
+        The views expressed here are the authors, and do not necessarily reflect the official views of the World Bank, its Executive Directors, or the countries they represent.
+      \end{lstlisting}
+    \item Confidentiality: YES or NO, or PARTIAL. \textcolor{red}{Do not add description}
+    \item Citation requirement: (empty)
+    \item License: as in scripts
```

Comment:
do we need to repeat this here?

Reply:
You have it in the data statement. But it is important information, so perhaps an isolated clear field has some value here.

Reply:
ok, agree there seems to be value in repeating